### ISSUE TITLE
Exclude autosaves and not yet reviewed versions from XLIFF export

### ISSUE THREAD

**Issue by karthik-integreat** [digitalfabrik/integreat-cms#1599]
### Describe the Bug

~While exporting XLIFFs, you have the choice to explicitly export only published sites. However it exports also automatically saved pages. These are not shown in the App and are not published sites. These should not be exported.~
When exporting all pages as XLIFF, also auto saved pages and those in review are included, even if only public versions and drafts should be included.

### Steps to Reproduce

1. Go to 'Deutschlandvorlage'
2. Click on 'Sites'. Change the Language in the drop down Menu to French. Apply the Filter "Translation is old" or "Translation does not exist"
3. Scroll down to 'the drop down menu'. Choose "Export all pages for the translations into French".
4. The Export is generated.

### Expected Behavior

The export should have all pages that do not have a translation or where the translation is old, but only if the status is either "PUBLIC" or "DRAFT", not "AUTOSAVE" or "IN_REVIEW".

Also add test data for those two status and make sure they don't end up in the exported test XLIFF files.

### Actual Behavior

In the export are automatically saved pages as well as those in review.

### Additional Information

![Bild eingefügt am 2022-7-18 08-38](https://user-images.githubusercontent.com/80322785/179461952-fc901a96-849e-4993-aef6-162aa9a6fe60.png)
<img width="914" alt="2022-07-18_09-19" src="https://user-images.githubusercontent.com/80322785/179461955-772ad652-0bb1-4f2e-a3a6-67f63fe4f59b.png">

**Comment by svenseeberg** [comment]
Indeed, version 19 is being exported my test. I do not yet understand why this is the case.
```
integreat=# SELECT language_id, version, status, creator_id, language_id, currently_in_translation FROM cms_pagetranslation WHERE page_id=10999 AND language_id=3 ORDER BY version ASC;
 language_id | version |  status   | creator_id | language_id | currently_in_translation 
-------------+---------+-----------+------------+-------------+--------------------------
           3 |       1 | PUBLIC    |            |           3 | f
           3 |       2 | PUBLIC    |            |           3 | f
           3 |       3 | PUBLIC    |            |           3 | f
           3 |       4 | PUBLIC    |            |           3 | f
           3 |       5 | PUBLIC    |            |           3 | f
           3 |       6 | PUBLIC    |        280 |           3 | f
           3 |       7 | PUBLIC    |        280 |           3 | f
           3 |       8 | PUBLIC    |        280 |           3 | f
           3 |       9 | PUBLIC    |        280 |           3 | f
           3 |      10 | PUBLIC    |         21 |           3 | f
           3 |      11 | PUBLIC    |        499 |           3 | f
           3 |      12 | PUBLIC    |        499 |           3 | f
           3 |      13 | PUBLIC    |        650 |           3 | f
           3 |      14 | PUBLIC    |        650 |           3 | f
           3 |      15 | PUBLIC    |        651 |           3 | f
           3 |      16 | PUBLIC    |        651 |           3 | f
           3 |      17 | PUBLIC    |        650 |           3 | f
           3 |      18 | PUBLIC    |        650 |           3 | f
           3 |      19 | AUTO_SAVE |        235 |           3 | f
```

AFAICT the query for selecting the pages is in [prefetched_public_translations_by_language_slug](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/models/abstract_content_model.py#L188), which filteres for `status=status.PUBLIC`.

**Comment by svenseeberg** [comment]
This was partially resolved in #1600
